src/components/DocumentViewer/DocumentViewer.tsx

```diff
@@ -1,40 +1,46 @@
-import React, { useState } from "react";
+import { obfuscateDocument, getData } from "@govtechsg/open-attestation";
+import { saveAs } from "file-saver";
+import React from "react";
 import { RecommendationsDisplay } from "../RecommendationsDisplay";
-import { findAllSensitiveFields } from "../SensitiveFieldsFinder";
 import { flatten } from "../shared";
 
 interface DocumentViewerProps {
-  document?: any;
+  wrappedDocument?: any;
```

Comment:
there is a type for this => WrappedDocument<any> from open attestation

Reply:
thanks for pointing it out! I was using WrappedDocument in PrivacyFilter, forgot about it here